docs/development/getting_started.rst

```diff
@@ -145,7 +145,7 @@ development, staging, and prod (production).
    hardening applied but virtualized
 
 If you plan to alter the configuration of any of these machines, make sure to
-review the `Development Guide for Serverspec Tests </docs/spec_tests.md>`__.
+review the `Development Guide for Serverspec Tests </docs/development/spec_tests.rst>`__.
```

Comment:
There should be no more expectation that this stuff works while browsing around Github.com. If it does, that's nice, but I'm not going to go out of my way to make that work. That was the old way. This is the new.